remote-workstations/new-agent-vm/server2016-graphics-agent.json

```diff
@@ -226,7 +220,7 @@
                     "osDisk": {
                         "name": "osdisk",
                         "vhd": {
-                            "uri": "[concat('http://',parameters('storageAccountName'),'.blob.core.windows.net/vhds/',parameters('dnsLabelPrefix'), 'disk0.vhd')]"
```

Comment:
I think we want the ability for the 'user' account (where binaries are stored) and the 'vhd' account to be separate, at least form the machine template standpoint. This will allow us to do multi-region deployments where the binaries are maybe only in one region but the VHD accounts are in different regions. I think I'd back out the changes in the machine templates and just point both parameters to the same account for now. that make sense?

Reply:
then how about create storage account in *-RW group and pass to template even for the admin-rw and used to hold VHD